tests/parsing/GlobbingTest.py

```diff
@@ -100,6 +101,41 @@ def test_alternatives(self):
                              sorted(alternatives))
 
 
+class EscapingTest(unittest.TestCase):
+
+    def _test_cases(self, in_arr, out_arr):
```

Comment:
Oh I see since you use this only once, maybe you can inline this, such statements are quite common in tests :) But that's you decision :+1: